src/Symfony/Component/BrowserKit/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+* Methods `AbstractBrowser::click()` and `AbstractBrowser::clickLink()` will have a new `$serverParameters` argument
+  in version 6.3, not defining it is deprecated
```

Comment:
Should the code trigger deprecations when that is the case?

Reply:
I did it that way to be consistent with a very similar change : https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/BrowserKit/CHANGELOG.md#420